### DISCUSSION TITLE
bypy.hashcache.json' not found, no caching

### DISCUSSION THREAD

**Discussion by Raywoo1** [houtianze/bypy#640] [Q&A]
Loading Hash Cache File '/home/xxx/.bypy/bypy.hashcache.json'...
Hash Cache File '/home/xxx/.bypy/bypy.hashcache.json' not found, no caching

请问这个情况咋办？

**Comment by houtianze** [comment]
可以不理，第一次运行没有哈希缓存文件而已，不影响正常使用。

**Reply by Raywoo1** [reply]
但是现在就一直卡在认证，让我等待300秒

**Reply by Raywoo1** [reply]
![image](https://user-images.githubusercontent.com/28223203/226284232-d53380c1-855d-4217-8c98-ec23d1dd2ffe.png)

**Reply by Raywoo1** [reply]
![image](https://user-images.githubusercontent.com/28223203/226287967-b4a8ed16-23b2-4e37-b0e8-c75d36be950a.png)
